apps/store/src/blocks/AccordionBlock.tsx

```diff
@@ -19,15 +20,14 @@ export const AccordionBlock = ({ blok }: Props) => {
           </StyledHeading>
         )}
         <div>
-          {blok.items.map((nestedBlock) => (
-            <StoryblokComponent key={nestedBlock._uid} blok={nestedBlock} />
+          {blocksOfType(blok.items, AccordionItemBlock.blockName).map((nestedBlock) => (
```

Comment:
nit: As an alternative, you could filter this before the return 
```suggestion
          const accordions = blocksOfType(blok.items, AccordionItemBlock.blockName)
          return (
            accordions.map((nestedBlock) =>
```

Reply:
fixed